dependencies/deployment/github/dependencies.bzl

```diff
@@ -0,0 +1,25 @@
+#
+# GRAKN.AI - THE KNOWLEDGE GRAPH
+# Copyright (C) 2018 Grakn Labs Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+
+def dependencies_for_github_deployment():
+    native.http_file(
+        name = "ghr",
+        executable = False,
```

Comment:
@lolski `executable = False` is already default, [src1](https://docs.bazel.build/versions/master/be/workspace.html#http_file_args), [src2](https://github.com/bazelbuild/bazel/blob/master/tools/build_defs/repo/http.bzl#L283-L284)